www/js/views/network-pools-form.js

```diff
@@ -58,6 +70,19 @@ module.exports = Backbone.Marionette.ItemView.extend({
         this.selectedUser = null;
     },
 
+    onNetworksChange: function () {
+        this.checkInput();
+        var data = Backbone.Syphon.serialize(this);
+        if (data.networks.length === 1 && (!this.selectedNetworks || this.selectedNetworks.length === 0)) {
+            this.selectedNetworks = data.networks;
+            this.render();
+        } else if (!data.networks) {
+            this.networks = this.allNetworks;
+            this.selectedNetworks = [];
+            this.render();
+        }
```

Comment:
yep, I need rerender view when select empty and I add network to it or after I remove the last one network from select.